server/src/routes/auth.routes.js

```diff
@@ -1,6 +1,6 @@
 const express = require("express");
 const authController = require("../controllers/auth.controller");
-const { validateSignupRequest, isRequestValidated } = require("../validators/auth.validator");
+const{ validateSignupRequest, isRequestValidated } = require('../validators/auth.validator')
```

Comment:
It's syntactically correct but based on the code style, there should be space between const and the variables then.